codebase/superdarn/src.doc/rfc/0023.rfc.xml

```diff
@@ -63,11 +63,11 @@
 <tr><td><pre> gvec={GridGVec, mlat:0.0, $      ; magnetic latitude
               mlon:0.0, $                    ; magnetic longitude
               azm: 0.0, $                    ; magnetic azimuth
-              vel: {GridValue,median:0.0, $  ; velocity mean
+              vel: {GridValue,median:0.0, $  ; velocity median
```

Comment:
As previously noted on the commit comments (https://github.com/SuperDARN/rst/commit/359d05182e28801ac0290022bb239ef0e18ea2c1#comments) and in #294, the grid velocity, power, and spectral width values are weighted means and not medians, so this change should probably be reverted.

Reply:
Hmmm, looks like we never reverted that commit before the last minor release. We should include additional explanation to prevent us from falling into this trap again.

Reply:
Can someone who knows what they are doing please revert this commit? e.g. @ksterne?

Reply:
Since 359d051 is tied up with another line addition, it's probably just easiest to modify them.
```suggestion
              vel: {GridValue,median:0.0, $  ; velocity weighted mean (it's known the parameter name is misleading)
```

Reply:
Didn't we put a big warning label on the main RFC page saying their contents may be inaccurate and/or out of date?  Would it be simpler just to revert the change and refer users to the new documentation instead?

Reply:
I agree that the best solution here is to remove the RFCs here that are out of date once we have better updated information.  That's a little bit much to be doing for the release branch and I think we can tackle that in 4.5.  It is possible to get to this information without going through the html documentation, so we might want to update it while our attention is on it and so that we remember it.  So, I'd say do this change for now so we don't forget about how it's misleading and in the future remove the RFC pages altogether when there's better documentation elsewhere.  Does that make sense @egthomas?

Reply:
I think I might have gotten this wrong on my suggestion.  @ecbland, do you think you could speak to this as I think @egthomas is out on a longer leave.  I think if I change my suggested commit that it's the velocity weighted median right?